drivers/dma/dma_mcux_edma.c

```diff
@@ -163,13 +167,15 @@ static void dma_mcux_edma_error_irq_handler(const struct device *dev)
 {
 	int i = 0;
 	uint32_t flag = 0;
+	struct dma_status status;
 
 	for (i = 0; i < DT_INST_PROP(0, dma_channels); i++) {
 		if (DEV_CHANNEL_DATA(dev, i)->busy) {
 			flag = EDMA_GetChannelStatusFlags(DEV_BASE(dev), i);
 			LOG_INF("channel %d error status is 0x%x", i, flag);
 			EDMA_ClearChannelStatusFlags(DEV_BASE(dev), i,
 						     0xFFFFFFFF);
+			dma_mcux_edma_get_status(dev, i, &status);
 			EDMA_AbortTransfer(DEV_EDMA_HANDLE(dev, i));
```

Comment:
Are we doing anything with the status field? If not then this change is not needed.

Reply:
ok, this is just for debug the status, let me remove it